c++/build/BUILD

```diff
@@ -0,0 +1,88 @@
+load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
```

Comment:
I think @kentonv wanted flags to be in src/kj since they are kj flags and not capnp flags.